doc/user-guide.md

```
@@ -0,0 +1,304 @@
+To try out the troubleshooting panel, you can deploy the [bad-app](https://github.com/korrel8r/demos/tree/main/bad-app) from the Korrel8r demos. It simulates a misbehaving application.
+
+```bash
+git clone https://github.com/korrel8r/demos.git
+cd demos/bad-app
+make deploy
+```
+
+The bad-app emits realistic error and info log messages every 0.5–2.5 seconds, then crashes after 30–90 seconds (randomized), entering `CrashLoopBackOff`. After a few minutes it will generate `KubePodCrashLooping`, `KubePodNotReady` alerts. 
+
+You can open the console and select the bad-app deployment manually, or jump to it like this:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)